### DISCUSSION TITLE
esp32 + pro micro

### DISCUSSION THREAD

**Discussion by moonbit5** [konwektor/BlueRetro#21] [General]
I've seen a post on Reddit which links to this repo. 

https://www.reddit.com/r/originalxbox/s/pobA8N2jR0

I've tried following the steps within the video mentioned but I've had limited success powering the esp32 via the pro micro. 

I can pair a controller if I power the esp32 directly, but the Bluetooth doesn't come up when powering via the Arduino. 

Do you have any more precise instructions for this setup? 

I believe the pro micro I'm using is already 3.3v.

Thanks!

**Comment by moonbit5** [comment]
I have the device working well now after using this: https://xgamesvideojuegos.blogspot.com/2023/02/mod-controles-bluetooth-en-xbox-og-v2.html 

do you know how I can update this setup to 1.9 + latest PSXtoXBOX firmware, I tried briefly but couldn't get an input. assume something needs tweaking on the blueretro side.

**Reply by konwektor** [reply]
Sorry but i do not know what You are trying to do - PSXtoXBOX adapter, or Blueretro+Ogx360?
With PSXtoXBOX try to ask David on his site.
For Blueretro+Ogx360 read  mine info: https://github.com/konwektor/BlueRetro/tree/master/!compiled
When something there is not clear for You, than come back here, and ask, I will try to help.
